packages/blockchain-utils-validation/src/blockchains/tezos.ts

```diff
@@ -0,0 +1,64 @@
+import { AccountID } from 'caip'
+import { RemoteSigner } from '@taquito/remote-signer'
```

Comment:
Wow, this package pulls in `libsodium`. That's stupid. What elliptic curve is Tezos using? I'm sure there's a better way to verify the signatures!

Reply:
They use many including p256, secp256k1, and ed25519.

Reply:
So it looks like this lib uses `elliptic` library for p256 and secp256k1 which is fine: https://github.com/ecadlabs/taquito/blob/eb6afed/packages/taquito-remote-signer/src/taquito-remote-signer.ts#L199

However the use of libsodium is not acceptable due to it's huge size: https://github.com/ecadlabs/taquito/blob/eb6afed/packages/taquito-remote-signer/src/taquito-remote-signer.ts#L188 when there exists much tinier libs such as this one: https://www.stablelib.com/modules/_ed25519_ed25519_.html#verify

Unfortunately I can not accept this PR as long as it's using libsodium. Maybe a PR to fix it in the taquito library is warranted?

Reply:
Isn't size of `blockchain-utils-validation` irrelevant? It's only used in `core` which doesn't really depend on size.

Reply:
@oed Does this help? https://github.com/ecadlabs/taquito/pull/978

Reply:
> @oed Does this help? ecadlabs/taquito#978
 Yes, that PR looks great!

Reply:
> Isn't size of blockchain-utils-validation irrelevant? It's only used in core which doesn't really depend on size.

It's not irrelevant. Ideally `core` would also be small enough to run in a browser, it's a tradeoff though. Don't want to just carelessly add more deps however.

Reply:
A step that should be taken outside of this PR would probably be removing any dependencies that use WebAssembly, since they are rather large for no noticable performance gain. That is why `sodium` is so large.